RecoTauTag/HLTProducers/src/L2TauTagNNProducer.cc

```diff
@@ -678,30 +662,18 @@ void L2TauNNProducer::fillPatatracks(tensorflow::Tensor& cellGridMatrix,
   auto getCell = [&](NNInputs input) -> float& {
     return getCellImpl(cellGridMatrix, tau_idx, phi_idx, eta_idx, input);
   };
+
+  std::vector<int> TrkGood;
+  std::vector<int> VtxGood;
+
+  selectGoodTracksAndVertices(patavtx_soa, patatracks_tsoa, TrkGood, VtxGood);
+
   const int nTaus = static_cast<int>(allTaus.size());
```

Comment:
```suggestion
  const int nTaus = allTaus.size();
```
Is `static_cast` actually needed, or useful here?